app/src/main/java/org/oppia/app/home/continueplaying/OngoingStoryViewModel.kt

```diff
@@ -0,0 +1,20 @@
+package org.oppia.app.home.continueplaying
+
+import android.view.View
+import androidx.lifecycle.ViewModel
+import org.oppia.app.model.PromotedStory
+
+// TODO(#297): Add download status information to promoted-story-card.
+
+/** [ViewModel] for displaying a promoted story. */
+class OngoingStoryViewModel(
+  val ongoingStory: PromotedStory,
+  private val ongoingStoryClickListener: OngoingStoryClickListener
+) : ContinuePlayingViewModel() {
+
+  val whiteBackground = 0xffffff
```

Comment:
This seems like it would never change. Can we use a color resource for this directly in the XML, instead?

Reply:
Removed this and introduced MaterialCardView library similar to #270